spec/linter-stylelint-spec.js

```diff
@@ -7,117 +7,136 @@ describe('The stylelint provider for Linter', () => {
 
   beforeEach(() => {
     atom.workspace.destroyActivePaneItem();
+    atom.config.set('linter-stylelint.usePreset', true);
+
     waitsForPromise(() => {
-      atom.packages.activatePackage('linter-stylelint');
-      return atom.packages.activatePackage('language-css').then(() =>
-        atom.workspace.open(path.join(__dirname, 'fixtures', 'good.css'))
-      );
+      return Promise.all([
+        atom.packages.activatePackage('linter-stylelint'),
+        atom.packages.activatePackage('language-css')
+      ]);
     });
-    atom.config.set('linter-stylelint.usePreset', true);
   });
 
-  describe('checks bad.css and', () => {
-    let editor = null;
-    beforeEach(() => {
-      waitsForPromise(() => {
-        return atom.workspace.open(path.join(__dirname, 'fixtures', 'bad', 'bad.css')).then(openEditor => {
-          editor = openEditor;
-        });
-      });
-    });
+  it('detects invalid coding style in bad.css and report as error', () => {
+    waitsForPromise(() => {
+      return atom.workspace.open(path.join(__dirname, 'fixtures', 'bad', 'bad.css')).then(editor => {
+        return lint(editor);
+      }).then(messages => {
+        expect(messages.length).toBeGreaterThan(0);
 
-    it('finds at least one message', () => {
-      waitsForPromise(() => {
-        return lint(editor).then(messages => {
-          expect(messages.length).toBeGreaterThan(0);
```

Comment:
Although the later tests failing would show something was wrong as well, so it isn't _necessary_, it just helps pin down what is going on.